.github/workflows/test_agents.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  pull_request: {}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - extra: ""
+            script: "test_basic.py"
+            python-version: "3.10"
+          - extra: "stable-baselines"
+            script: "test_sb.py"
+            python-version: "3.7"
+          - extra: "stable-baselines3"
+            script: "test_sb3.py"
+            python-version: "3.10"
+          - extra: "ray-rllib"
+            script: "test_ray_rllib.py"
+            python-version: "3.10"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{matrix.python-version}}
```

Comment:
@discordianfish I needed a different python version for them, as SB requires 3.7. I handled it this way.

In addition, we are limited to python 3.10 for rllib (and I believe also others). 

Is this ok how it is handled here?